matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/model/relation/RelationService.kt

```diff
@@ -153,5 +153,5 @@ interface RelationService {
      * from the backend
      * @param rootThreadEventId the root thread eventId
      */
-    suspend fun fetchThreadTimeline(rootThreadEventId: String): List<Event>
+    suspend fun fetchThreadTimeline(rootThreadEventId: String): Boolean
```

Comment:
Would be nice to document what is the meaning of a returned value `false` or `true`.

Reply:
Thanks, this code is not yet in use.  I will refactor it completely so I will update the documented accordingly